db/migrations/20240531130301_Pages.sql

```diff
@@ -0,0 +1,10 @@
+-- Ensure the UUID extension is enabled
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
Since this was in the first migrations we can probably leave it off now!

Reply:
Do u want me to delete it?

Reply:
Awesome! Done